google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -44,6 +44,24 @@
       </p>
       <br/>
 
+      <h2>[17.6.1]</h2>
+      <h3>Added</h3>
+      <ul>
+        <li>App Engine flexible facet with app.yaml and Dockerfile configuration. (<a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1514">#1514</a>)</li>
+        <li>App Engine flexible framework support detection. (<a href="https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/1277">#1277</a>)</li>
```

Comment:
The link is broken, did you mean: https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1277